### PR TITLE
chore: add doc about v1 vdm release

### PR DESCRIPTION
**Pull request by jjtang1985**
This PR contains the alignment of potential v1 vdm release.

### CODE REVIEW

File: knowledge-base/implementation-documentation/0010-release-version-1-vdms.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## When releasing 1.0 VDMs should happen
+
+Only if when our generated code contains vulnerable code, we need to fix the generator and release new clients.
+This should not happen, and should be treated as edge case.
+When necessary, please avoid the following issue:
+
+- when publishing new VDMs for 1.0, the `latest` tag should still point to version 2. Maybe talking to the assembly team for additional information as they handle the bom PR. The worst case would be we release version 1 VDMs followed by version 2 VDMs, as a work around.
+
+## How to plan the support of releasing 1.0 VDMs
```
Comment (florian-richter) [inline knowledge-base/implementation-documentation/0010-release-version-1-vdms.md:14]: I dont understand this heading. Is this meant as a preemptive thing or a thing that needs to be done when the edge case happens?

Comment (jjtang1985) [reply knowledge-base/implementation-documentation/0010-release-version-1-vdms.md:14]: Edge cases cannot be 100% avoided, in general.
In the previous meeting, there was a concern about the potential version 1 VDM release.
As a developer, the decision might not be under our control.
Therefore, I added a section about some processes and dependencies.

Comment (florian-richter) [reply knowledge-base/implementation-documentation/0010-release-version-1-vdms.md:14]: Sorry, but I still don't get if this section should be a "In the unlikely case of emergency, follow these steps" or rather "Preparations that need to be done now so we are prepared if something happens".

**Review by florian-richter — APPROVED** [review]
clarified in call

**Comment by jjtang1985** [comment]
@artemkovalyov , as discussed, please review.

File: knowledge-base/implementation-documentation/0010-release-version-1-vdms.md (hunk 1 above)
Comment (jjtang1985) [reply knowledge-base/implementation-documentation/0010-release-version-1-vdms.md]: Discussed with @marikaner , let's keep this internally.

**Pull request merged**